### PR TITLE
DIA-2440 category details view in manage preferences

### PR DESCRIPTION
**Pull request by Nevazhnovu**

### CODE REVIEW

File: ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/GDPR/GDPRPMConsentSnapshot.swift [hunk 1]
```diff
@@ -64,7 +64,7 @@ class GDPRPMConsentSnaptshot: NSObject, ConsentSnapshot, PMVendorManager, PMCate
         if disclosureCategories.isNotEmpty() {
             for category in disclosureCategories {
                 self.specialPurposes.insert(
-                    GDPRCategory(name: category.name, description: category.description, disclosureOnly: true, legIntVendors: category.disclosureOnlyVendors)
+                    GDPRCategory(name: category.name, description: category.description, disclosureOnly: true, legIntVendors: category.disclosureOnlyVendors, requiringConsentVendors: category.disclosureOnlyVendors)
```
Comment (andresilveirah) [inline ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/GDPR/GDPRPMConsentSnapshot.swift:67]: @Nevazhnovu is that correct?
```swift
 requiringConsentVendors: category.disclosureOnlyVendors
 ```

Comment (Nevazhnovu) [reply ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/GDPR/GDPRPMConsentSnapshot.swift:67]: @andresilveirah yes

Comment (Nevazhnovu) [reply ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/GDPR/GDPRPMConsentSnapshot.swift:67]: Disclosure Only Vendors exist in both tabs in the web solution so the above is correct

Comment (andresilveirah) [reply ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/GDPR/GDPRPMConsentSnapshot.swift:67]: @Nevazhnovu but as the name suggests, Disclosure Only Vendors, are vendors that don't require consent. They can't be toggled on or off.

Comment (Nevazhnovu) [reply ConsentViewController/Classes/Views/tvOS/NativePrivacyManager/GDPR/GDPRPMConsentSnapshot.swift:67]: @andresilveirah Disclosure Only Vendors don't have any buttons.

**Review by andresilveirah — APPROVED** [review]

**Pull request merged**